net/chrony/src/opnsense/mvc/app/models/OPNsense/Chrony/General.xml

```diff
@@ -0,0 +1,31 @@
+<model>
+    <mount>//OPNsense/chrony/general</mount>
+    <description>Chrony configuration</description>
+    <version>0.0.1</version>
+    <items>
+        <enabled type="BooleanField">
+            <default>0</default>
+            <Required>Y</Required>
+        </enabled>
+        <port type="PortField">
+            <default>323</default>
```

Comment:
typo? ntp is afaik 123

Reply:
The default for chrony is 323 to not conflict with existing installations.